src/util.h

```diff
@@ -382,4 +382,49 @@ debug_hex_dump_slice(platform_log_handle *, uint64 grouping, slice data);
     : ((intval) < 1000) ? "3d"                                                 \
                         : "4d")
 
+// Size of on-stack char message buffer (mostly used in diagnostics)
+#define DIAG_MSGBUF_SIZE 80
+
+// Convenience specifier for output buffer to convert size with unit specifier
+#define SIZE_TO_STR_LEN 20
+
+// Min size of buffer needed if size is enclosed in (), as : '(~3.50 GiB)'
+#define SIZE_TO_STR_MINLEN (SIZE_TO_STR_LEN - 2)
```

Comment:
This is used only in one  debug_assert in size_to_str.

Furthermore, I don't think that debug_assert even makes sense.  Why is there room for parens required in these buffers?  size_to_str does not put parens in its output buffers.

Reply:
You are right. This is left-over cruft from some versions of code I wrote earlier, and is now no longer needed. 
Deleted this `SIZE_TO_STR_MINLEN` define.

I reworked the assertion in `size_to_str()` to check that the `outbuflen >= SIZE_TO_STR_LEN`.